portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Nikita's Paintings Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Nikita's Paintings Gallery</h1>
+      <div class="image-thumbnails">
+        <a href="images/Painting1.jpeg"><img src="images/Painting1.jpeg" style="width:900px;height:900px;"/></a>
```

Comment:
Suggest to use CSS class here instead of inline CSS, so that the same class can be re-used.

Reply:
Changed the whole file.